docs/concepts/extra-attributes.md

```diff
@@ -52,6 +52,24 @@ const paragraphExtension = new ParagraphExtension({
 
 This example accomplishes the same things as the previous example and remirror is smart enough to automatically parse the dom and write to the dom the required values.
 
+### Render as data attributes
+
+You can return an array of a key value pair, to determine how your extra attribute is rendered in the DOM
```

Comment:
```suggestion
You can return an array of a key value pairs, to determine how your extra attribute is rendered in the DOM.
```

Reply:
I would read "pairs" as though you could pass multiple? I.e.

```
[
  ['key1', 'val1'],
  ['key2', 'val2']
]
```

Reply:
Yes you're right. Ignore this.